dashboard/index.html

```diff
@@ -0,0 +1,231 @@
+                                                    <tr class="divide-x divide-gray-400" :class="classFromPosition(position)"> 
+                                                        <td class="p-1 border-b border-gray-400">{{ position.id }}</td>
+                                                        <td class="p-1 border-b text-center uppercase">{{ position.status }}</td>
+                                                        <td class="p-1 border-b">{{ formatTime(position.signal_tstamp) }}</td>
+                                                        <td class="p-1 border-b">{{ position.amount }}</td>
+                                                        <td class="p-1 border-b">{{ formatPrice(position.wanted_entry) }}</td>
+                                                        <td class="p-1 border-b">{{ formatPrice(position.initial_stop) }}</td>
+                                                        <td class="p-1 border-b">{{ formatTime(position.entry_tstamp) }}</td>
+                                                        <td class="p-1 border-b">{{ formatPrice(position.filled_entry) }}</td>
+                                                        <td class="p-1 border-b">{{ formatPrice(position.currentStop) }}</td>
+                                                        <td class="p-1 border-b" :class="classFromResult(position.worstCase)">{{ formatResult(position.worstCase) }}</td>
+                                                        <td class="p-1 border-b">{{ formatPrice(position.initialRisk) }}</td>
+                                                    </tr>
+                                                </template>
+                                            </template>
+                                        </tbody>    
+                                    </table>
+                                </div>
+                            </div>
+                        </template>
+                    </div>
+                </div>
+            </div>
+        </main>
+    </div>
+
+
+    <script>
+        const Dashboard = {
+            data() {
+                return {
+                    view: 'dashboard',
+                    viewInitialized: false,
+                    dashboard: {
+                        isLoading: true,
+                        bots: [ ],
+                    }
+                }
+            },
+            computed: {
+                classForLoading() {
+                    return {
+                        'animate-spin': this.dashboard.isLoading
+                    }
+                },
+            },
+            mounted() {
+                this.viewInitialized = true
+                this.loadDashboardData()
+            },
+            methods: {
+                loadDashboardData() {
+                    this.dashboard.isLoading = true
+                    window.fetch('/dashboard/dashboard.json?v=' + Date.now())
```

Comment:
would prefer to keep it on the same level as the index.html or at least relative to the index.html and not absolute.
 makes it more flexible to be used on different production-server setups

Reply:
No problem with that - will update.